### PR TITLE
Changes for v8.3.0 - helm - WIP

### PR DESCRIPTION
**Pull request by elnyry**
## Overview of Helm changes:
- Replacing port 50051 with 50055 as per - https://github.com/mojaloop/project/issues/1015#issuecomment-544926397

## Maintenance:
- Updated DB config for below services in PR #215
  - ALS
  - central-ledger
  - quoting-service

## Releases
- ml-api-adapter: v8.3.0
- central-ledger: v8.3.0
- account-lookup-service: v8.3.0
- quoting-service: v8.3.2-snapshot
- central-settlement: v8.3.0
- central-event-processor: v8.2.0
- bulk-api-adapter: v8.1.0-snapshot
- email-notifier: v7.3.0
- simulator: v8.1.1

### CODE REVIEW

File: centralledger/chart-handler-admin-transfer/values.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-admin-transfer/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-handler-timeout/values.yaml [hunk 2]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-timeout/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-handler-transfer-fulfil/values.yaml [hunk 3]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-transfer-fulfil/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-handler-transfer-get/values.yaml [hunk 4]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-transfer-get/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-handler-transfer-position/values.yaml [hunk 5]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-transfer-position/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-handler-transfer-prepare/values.yaml [hunk 6]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-handler-transfer-prepare/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: centralledger/chart-service/values.yaml [hunk 7]
```diff
@@ -69,7 +69,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline centralledger/chart-service/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: ml-api-adapter/chart-handler-notification/values.yaml [hunk 8]
```diff
@@ -37,7 +37,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline ml-api-adapter/chart-handler-notification/values.yaml]: this should not be changed please revert and do the same in the event-sdk

File: ml-api-adapter/chart-service/values.yaml [hunk 9]
```diff
@@ -37,7 +37,7 @@ sidecar:
     periodSeconds: 15
   config:
     event_logger_grpc_host: localhost
-    event_logger_grpc_port: 50051
+    event_logger_grpc_port: 50055
```
Comment (rmothilal) [inline ml-api-adapter/chart-service/values.yaml]: this should not be changed please revert and do the same in the event-sdk

**Comment by elnyry** [comment]
@rmothilal never mind

**Pull request closed without merging**